utils/mount/overlay2.go

```diff
@@ -46,15 +45,15 @@ func NewMountDriver() Interface {
 	return &Default{}
 }
 
-func supportsOverlay() bool {
```

Comment:
Hang on a second.
I think there are two condition to detect the overlay support.
"modprobe overlay" || "/proc/filesystems".
I'm just afraid that some of cases that there will be false in "/proc/filesystems" case.